website/src/pages/index.js

```diff
@@ -28,14 +28,14 @@ const useWindowSize = () => {
 const ThemeResetComponent = () => {
   const {isDarkTheme, setLightTheme, setDarkTheme} = useThemeContext();
 
-  useEffect(() => {    
+  useEffect(() => {
     const children = document.querySelector(".navbar__items--right").childElementCount;
     document.querySelector(".navbar__items--right").childNodes[children-2].style.display = "none";
 
     if(isDarkTheme) {
```

Comment:
How about using `setLightTheme(!isDarkTheme)`?

Reply:
Do you mean：
```diff
- if(isDarkTheme){
-   setLightTheme(true);
- }
+ setLightTheme(!isDarkTheme)
```
I have no plans to change this part of the code. 😳
But, I'll try it.
I think this might make the function execute more times. I'll check it right way.

Reply:
@juzhiyuan Sorry, I made a mistake. The intent of this part of the code is to reset the color mode to `light mode`. The above changes will not achieve this goal

If you mean, change `setLightTheme(true)`  to `setLightTheme(!isDarkTheme)`. I think it might be clearer to use `ture` as a parameter.

Reply:
@juzhiyuan so I don't think it should be changed. 😅